src/AccessibilityInsights.SharedUx/Resources/Styles.xaml

```diff
@@ -113,7 +113,7 @@
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="{x:Type Button}">
-                    <Border Background="{TemplateBinding Background}">
+                    <Border Background="{TemplateBinding Background}" BorderBrush="{TemplateBinding Foreground}" BorderThickness="{DynamicResource ResourceKey=ActionsButtonBorderThickness}" CornerRadius="3" Padding="4,1">
```

Comment:
The template will override any value set in the xaml file, so I moved the padding here to keep related things together.

Reply:
Looking at this, the thought crosses my mind: is there something important the BtnActions style is doing with the ControlTemplate that BtnStandard isn't? BtnStandard​​​​​​​ already provides a border in HC modes. Are those new IsMouseOver colors importantly different vs BtnStandard's?

Reply:
The brush colors differ only in dark mode. There, BtnActions needs to have a lighter IsMouseOver color than BtnStandard because BtnActions is sometimes displayed against a highlighted background. I tried using the colors from BtnStandard and the change on mouse hover was almost imperceptible. I'll proceed with this change, but will investigate the possibility of using the lighter IsMouseOver color for BtnStandard. We'd still need the styling inside the ControlTemplate, but we might be able to consolidate the brushes.